src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -42,26 +41,26 @@ public EditCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditCommand.MESSAGE_USAGE), pe);
         }
 
-        EditPersonDescriptor editPersonDescriptor = new EditPersonDescriptor();
+        EditCommand.EditTuteeDescriptor editTuteeDescriptor = new EditCommand.EditTuteeDescriptor();
```

Comment:
Perhaps you can include `import seedu.address.logic.commands.EditCommand.EditTuteeDescriptor;` above?